File: Pratica 6 - Funcoes 2/cpp_logic/exercicio10_v2.cpp
#include <iostream>
using namespace std;

void tabuada(int numero) {
    for (int i = 1; i <= 10; i++) {
        cout << numero << " x " << i << " = " << numero * i << endl;
    }
}

int main() {
    int numero;
    cout << "Digite um número inteiro: ";
    cin >> numero;

    tabuada(numero);
    return 0;
}

File: Pratica 6 - Funcoes 2/java_logic/Exercicio10_v2.java
import java.util.Scanner;

public class Exercicio10_v2 {
    static void tabuada(int numero) {
        for (int i = 1; i <= 10; i++) {
            System.out.println(numero + " x " + i + " = " + (numero * i));
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite um número: ");
        int numero = sc.nextInt();
        tabuada(numero);

        sc.close();
    }
}
